lib/web_ui/lib/src/engine/platform_views/content_manager.dart

```diff
@@ -223,3 +240,18 @@ class PlatformViewManager {
     return result;
   }
 }
+
+DomElement _defaultFactory(
+  int viewId, {
+  Object? params,
+}) {
+  params!;
+  params as _DefaultFactoryParams;
+  return domDocument.createElement(params.tagName);
+}
+
+typedef _DefaultFactoryParams = Map<Object?, Object?>;
```

Comment:
```suggestion
typedef _DefaultFactoryParams = Map<Object, Object?>;
```

Reply:
This is because the type sent from `HtmlElementView` is `Map<dynamic, dynamic>`, and doing `Map<dynamic, dynamic> as Map<Object, Object?>` fails.

But this is a good idea. I'll change it on the framework to `Map<Object, Object?>` instead.

Reply:
Hmm I actually have a feeling this isn't going to work but I'll try it anyway.

`creationParams` go through a platform message and get serialized & deserialized, losing all their type information. So they are deserialized back as a `Map<dynamic, dynamic>`.

Reply:
Yep, this didn't work. Even if I change `HtmlElementView` to send `creationParams` as `Map<Object, Object>`, it arrives to the engine as `Map<dynamic, dynamic>` (because of serialization & deserialization of platform messages).

Reply:
Ah yes, `dynamic` is bad, and makes sense why you had to substitute by `Object?`... And I guess there's nothing "wrong" in using `null` as a possible key for a Map :/